pages/order-eggs.tsx

```diff
@@ -0,0 +1,12 @@
+const OrderEggs = () => {
+  return (
+    <>
```

Comment:
nit: extra brackets

Reply:
just kidding, we need the outside brackets here to wrap the multiple child elements

Reply:
use `<React.fragment>`